### ISSUE TITLE
Improve logging system

### ISSUE THREAD

**Issue by dostrelith678** [cardano-community/koios-artifacts#54]
**Is your feature request related to a problem? Please describe.**
There are 2 problems I see with Koios-specific log files stored on the nodes:
1. They are not well structured (besides `grest-poll.sh` logs, the others are all appended to their respective single file). Inside those logfiles, the structure is sometimes hard to navigate as `psql` output intersects the cron script output.

2. They do not have an automatic rotation system.

**Describe the solution you'd like**
Some ideas:
1. Separate individual log files per time interval (daily, weekly or monthly) and improve logging inside the cron scripts.
2. A cron log clean-up job to keep only the last 30 days' worth of data?

**Comment by rdlrt** [comment]
We could adapt a common format such as below (where `GREST_CRON_LOG` would go in `env`):
```bash
function logstr() {
  local dt fname loglvl msg
  dt=$(date -Is)
  fname=$(basename $0| cut -d. -f1)
  loglvl="${1}"
  msg="${2}"
  echo -e "${dt}\t- ${fname} - ${loglvl} - ${msg}" >> "${LOG_DIR}"/"${GREST_CRON_LOG}"_"$(date +%d%m%y)"
}

logstr INFO "This is a test message"
```

The output should look like below:
```
cat grest-cron_290722 
2022-07-29T07:12:52+00:00       - cnode-testlogger - INFO - This is a test message
2022-07-29T07:14:15+00:00       - cnode-testlogger - INFO - This is a test message
```

Again, archival would be up to user, as some might want/not want the old data, can even leave commented `find` command or add a param in case someone would like to have cleanup done automatically)

**Comment by dostrelith678** [comment]
That looks good to me.

**Comment by rdlrt** [comment]
Closed in 1.0.10rc
